### PR TITLE
Change virtual environment name on README

### LINKED ISSUE #459: Wrong paths in README setup
**Describe the bug**
Some of the steps in Project bootstrap section point to old references.

```diff
- Add an email address to the ADMINS settings variable in {{project_name}}/{{project_name}}/settings/base.py
+ Add an email address to the ADMINS settings variable in {{project_name}}/backend/{{project_name}}/settings/base.py
- Change the SERVER_EMAIL to the email address used to send e-mails in {{project_name}}/{{project_name}}/settings/production.py
+ Change the SERVER_EMAIL to the email address used to send e-mails in {{project_name}}/backend/{{project_name}}/settings/production.py
```

### PR DESCRIPTION
**Pull request by vanessa**
## Description

* Change virtual environment names when using virtualenvwrapper or normal virtualenv package
* Fix paths to `SERVER_EMAIL` and `ADMINS` change step (closes #459)

## Motivation and Context

* When using virtualenvwrapper, the created virtual environment isn't added to the root directory, but rather in a directory in `$HOME`, so we don't have to worry about it being added to source control and what name it is, so it's ok to have the virtualenv with the same name as project's.
* When using Python's virtualenv package, the directory is created at root level and should be the name of what we add to `.gitignore` and linter configs so that the files inside it are ignored from any processes we run. That's why it's important to instruct the user to name it `venv`.
* If someone creates a virtualenv with the name of the project, as instructed by the setup steps, it'll end up being added to source control, so if the person has CircleCI integrated, the linters will run and catch some problems inside the virtual environment folder. That's not something we want. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires documentation updates.
- [x] I have updated the documentation accordingly.
- [ ] My change requires dependencies updates.
- [ ] I have updated the dependencies accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,15 +31,15 @@ django-admin startproject theprojectname --extension py,yml,json --name Procfile
 ```
 In the next steps, always remember to replace theprojectname with your project's name
 - [ ] Above: don't forget the `--extension` and `--name` params!
-- [ ] Navigate to the project's directory through your command line.
-- [ ] Create a new virtualenv with either [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/) or only virtualenv: `mkvirtualenv theprojectname` or `python -m venv theprojectname`.
-- [ ] Make sure the virtualenv is activated `workon theprojectname` or `source theprojectname/bin/activate`
-- [ ] Install pip-tools if not installed yet: `pip install pip-tools` (maybe you'll have to run this command as an OS superuser).
-- [ ] Make sure you have Python 3.7 installed.
+- [ ] Navigate to the project's directory through your command line
+- [ ] Create a new virtualenv with either [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/) or only virtualenv: `mkvirtualenv {{project_name}}` or `python -m venv venv`.
```
Comment (davidpierre21) [inline README.md]: You missed one period left here :smile: 
```suggestion
- [ ] Create a new virtualenv with either [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/) or only virtualenv: `mkvirtualenv {{project_name}}` or `python -m venv venv`
```

Comment (vanessa) [reply README.md]: Thanks!

**Comment by vanessa** [comment]
@davidpierre21 You mean adding something like `{{project_name}}/` to .gitignore? So when the user start the project and name it `theprojectname` AND create the virtualenv with the same name, it'll be ignored, right?

**Comment by davidpierre21** [comment]
@vanessa Yes, something like that. Maybe a prefix or suffix, like {{project_name}}-venv or something like this, so it wouldn't ignore the project directory itself, if it makes any sense. Although using something like `venv` is way simpler, for someone who's nor familiar with virtual environments, it might conflict with some other existing environment. What do you think? :smiley:

**Pull request merged**